src/codegen.zig

```diff
@@ -80,8 +80,8 @@ pub fn generateSymbol(
     switch (typed_value.ty.zigTypeTag()) {
         .Fn => {
             switch (bin_file.options.target.cpu.arch) {
-                .wasm32 => unreachable, // has its own code path
-                .wasm64 => unreachable, // has its own code path
+                .wasm32 => return Function(.wasm32).generateSymbol(bin_file, src, typed_value, code, debug_output),
+                .wasm64 => return Function(.wasm64).generateSymbol(bin_file, src, typed_value, code, debug_output),
```

Comment:
wasm64 is currently vaporware and this should remain unreachable until there exists a runtime with support for it.